core/utils/ethabi.go

```
@@ -232,6 +232,11 @@ func Bytes4FromString(s string) [4]byte {
 	return b4
 }
 
+func MustAbiType(ts string, components []abi.ArgumentMarshaling) abi.Type {
+	ty, _ := abi.NewType(ts, "", components)
```

Would it be better to panic on error than silently ignore?